### PR TITLE
Renamed namespace

### LINKED ISSUE #3: Rename the namespace
The files currently still use the `JakubOnderka` top-level namespace. To prevent issues, it would probably be a good idea to switch this over to using `PHP_Parallel_Lint` or `PHPParallelLint`.

### PR DESCRIPTION
**Pull request by grogy**
It resolve #3

### CODE REVIEW

**Comment by grogy** [comment]
It is a candidate for release 1.0 version. I don't want to create 0.x with BC break. I mean the name `PHPParallelLint` of the namespace is best. It will be same in other packages.

Does it sense to @jrfnl?

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Internal
 
+- BC-Break: Changed - renamed namespace [#10](https://github.com/php-parallel-lint/PHP-Console-Color/pull/10) [@grogy](https://github.com/grogy).
```
Comment (jrfnl) [inline CHANGELOG.md]: I'd suggest moving the changelog entry up into a `### Changed` header. This is **_not_** an _internal_ change, but a change to the Public API, so should be annotated as such.

File: tests/ConsoleColorTest.php [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?php
-use JakubOnderka\PhpConsoleColor\ConsoleColor;
+use PHPParallelLint\PhpConsoleColor\ConsoleColor;
```
Comment (jrfnl) [inline tests/ConsoleColorTest.php]: Would it make sense to add a namespace to this test file as well ? I'm thinking `PHPParallelLint\PhpConsoleColor\Tests` would be in line with PSR-4 ?

**Review by jrfnl — COMMENTED** [review]
> I don't want to create 0.x with BC break.

[Semver](https://semver.org/)-wise, BC-breaks in 0.x versions are allowed, though it would need a new minor, i.e. `0.4`.

Composer treats minors below `1.0` as majors anyway, so it wouldn't break anything unless someone doesn't use proper version constraints, like `*`, but then `1.0` would break their package just the same.

Having said that, if you feel the package is mature enough for `1.0`, go for it! I'm all for stable release packages :+1: 

> I mean the name PHPParallelLint of the namespace is best. It will be same in other packages.

As that's in line with the GH organization name, that seems a logical choice. Or, as the GH organisation name has dashes, possibly `PHP_Parallel_Lint`.

The only thing which looks a bit weird is the PHP acronym being in uppercase in the top-level name, but in mixed case in the next level... should that be made consistent ?

```php
namespace PHPParallelLint\PhpConsoleColor;
//        ---             ---
```

File: CHANGELOG.md (hunk 1 above)
Comment (grogy) [reply CHANGELOG.md]: You have the right

**Comment by grogy** [comment]
Thanks for the link, I copy from semver: `MAJOR version when you make incompatible API changes`

I mean that the namespace is API for programmers with this library. For it, I want upgrade major version.

What about create 0.4 version [without this PR](https://github.com/php-parallel-lint/PHP-Console-Color/blob/master/CHANGELOG.md#unreleased) and 1.0 with only this PR? 

----

Namespace inconsistence - oh, I am missing it. Version with underscores will be better.

**Comment by jrfnl** [comment]
> What about create 0.4 version without this PR and 1.0 with only this PR?

In that case, I would recommend going straight to `1.0`. There is nothing currently in "unreleased" that affects end-users of this repo, so having a `0.4` with just those changes, adds no value.

**Comment by grogy** [comment]
Thanks for your feedback. So I will create 1.0 after merge this PR.

Namespace to tests is added ;)

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Changed
+
+- BC-Break: Changed - renamed namespace [#10](https://github.com/php-parallel-lint/PHP-Console-Color/pull/10) [@grogy](https://github.com/grogy).
```
Comment (jrfnl) [inline CHANGELOG.md]: To make it more obvious what has changed, you may want to mention the "from" and the "to".

> The top-level namespace for all classes has changed from `JakubOnderka` to `PHP_Parallel_Lint`.

**Review by jrfnl — COMMENTED** [review]
Left one minor remark, other than that, looks good to go.

File: CHANGELOG.md (hunk 3 above)
Comment (grogy) [reply CHANGELOG.md]: You have right, thanks

**Review by jrfnl — APPROVED** [review]

**Pull request merged**